force-app/main/default/classes/OmdbConnectionTest.cls

```diff
@@ -0,0 +1,207 @@
+@isTest
+private class OmdbConnectionTest {
+    // TODO ADD SYSTEM ASSERTS
```

Comment:
I would delete unused functions or large comment blocks very rarely do you need them and they clutter up a class and kind of stick around forever. With version control, we can bring them back if we need to so there isn't a reason to keep them commented out